lib/ip_blocklist.js

```diff
@@ -146,6 +146,10 @@ module.exports = function (log, config) {
 
         return result
       })
+
+      return {
+        found: found
+      }
```

Comment:
I don't really understand the change to return an object here - is it just for symmetry with the same method in the manager class?